src/inmanta/config.py

```diff
@@ -445,6 +447,80 @@ def __init__(self, name: str, port: int = 8888) -> None:
 # auth
 #############################
 AUTH_JWT_PREFIX = "auth_jwt_"
+AUTH_JWT_CLAIM_RE = r"^\s*([^\s].+)\s+(in|is)\s+([^\s].+)\s*$"
+
+claim_type = dict[str, str | list[str]]
+
+
+@dataclasses.dataclass
```

Comment:
All of the auth config could be moved into a separate module or in the server config?

Config is loaded very early, so we do have to be aware for import loops.

Reply:
seems to work

Reply:
Not sure how I will be able to move this into inmanta.protocol without import problems. See CI